src/components/common/GoogleButton.js

```diff
@@ -20,6 +20,7 @@ const GoogleButton = (props) => {
         </button>
       )}
       buttonText="Login with Google"
+
```

Comment:
Why are we adding this space here?

Reply:
@KvNGCzA feedback well received.
@KvNGCzA done.